ansible/roles/azure/tasks/create_multiple_vmss.yml

```diff
@@ -30,6 +30,23 @@
     file: "{{ deploy_path }}/conf/azure_multiple_vmss_vars.yml"
     name: azure_multiple_vmss_vars
 
+# Minimum check to ensure azure_multiple_vmss_vars.yml contains the right set of services
+- name: Fail when HA is not enabled and azure_multiple_vmss_vars.yml contains HA services
+  fail:
+    msg:  |
+          Incorrect roles in azure_multiple_vmss_vars.yml, 'journalnode' or 'zkfc' required only when HA is enabled.
+          Refer the VMSS definition in the example yaml file.
+  with_items: "{{ azure_multiple_vmss_vars.vars_list }}"
+  when: "('journalnode' in item.roles or 'zkfc' in item.roles) and not hdfs_ha"
+
+- name: Fail when HA is enabled and azure_multiple_vmss_vars.yml does not contain HA services
```

Comment:
I'm sorry if I missed something, but why introduce these validations here separately from the rest of the validations in the azurevalidations.py file?

Reply:
Addressed this in [1c5c08f](https://github.com/apache/fluo-muchos/pull/395/commits/1c5c08f3d1f2367e094f8e9ec9ea02c759658408)